README.md

```diff
@@ -124,10 +124,19 @@ Make sure you have Python 3.10 or 3.11 installed.
 (check out [`application/core/settings.py`](application/core/settings.py) if you want to see more config options.)
 
 2. (optional) Create a Python virtual environment:
```

Comment:
I would just make `Python virtual environment` on this line a link to the official documentation

Reply:
should i remove the command on both windows and other platforms ?

Reply:
Done !!

Reply:
No, I don't thnk so.

Reply:
wait let me do it again

Reply:
sir can you please review the change ?

Reply:
My comment was only about this documentation comment. And related to the newly added documentation line. I suggest adding a link to already existing text in line 126.

Reply:
done :)